tests/testthat/examples/compare_simple.cpp

```diff
@@ -1,11 +1,12 @@
 // [[odin.dust::compare_data(observed = double)]]
+// [[odin.dust::compare_data(another = int)]]
```

Comment:
`another` is ignored/not used in this compare function?

Reply:
yeah, I needed a second data element in order to trigger and fix a bug with multiple data elements (the constructor was put together without commas!). It's not used here though because the compare function is so stupid